roles/bwc/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+bwc_pkg_repo: "enterprise"
+# 'latest' to get latest version or numeric like '2.1.1'
+bwc_version: latest
+# used only if 'bwc_version' is numeric
+bwc_revision: 1
+
```

Comment:
Please define all vars used in this role with comments, as well as include them in `README.md`.